### ISSUE TITLE
Using pySym

### ISSUE THREAD

**Issue by nguyenthanhvuh** [bannsec/pySym#2]
Hi, Is there a way to use pySym directly, e.g., I can import pyPath, ast, directly to my Python code,  instead of having to use pip or docker?   When using pip, I cannot get ipython to work properly (error about `ImportError: No module named 'encodings'`.   I am using Debian stretch.

**Comment by bannsec** [comment]
I don't believe that has to do with pySym. Sounds like it's either python library confusion or perhaps a wrong python version.

Have you tried installing it into a virtualenv?

```bash
virtualenv --python=$(which python3) pySym && . pySym/bin/activate && pip install ipython .
```

Otherwise, upload/copy paste the full error.
